File: yukicoder/yuki9002.go
package main

import (
	"fmt"
)

func main() {
	var n, m int
	fmt.Scan(&n)
	for n < m {
		switch {
		case n%15 == 0:
			fmt.Println("Fizz")
		case n%3 == 0:
			fmt.Println("FIZZ!")
		case n%5 == 0:
			fmt.Println("BUZZ!")
		default:
			fmt.Println(n)
		}
		n++
	}

}

File: yukicoder/yuki9002.exs
defmodule Main do
  def main do
    fizzbuzz = fn(a) ->
      cond do
        Integer.mod(a, 15) == 0 -> "FizzBuzz"
        Integer.mod(a, 3)  == 0 -> "Fizz"
        Integer.mod(a, 5)  == 0 -> "Buzz"
        :else                   -> a
      end
    end
    a = IO.gets("") |> String.trim() |> String.to_integer()
    for a <- 1..a do
      IO.puts fizzbuzz.(a)
    end
  end
end

Main.main